File: dws-step/src/main/java/io/dws/step/config/DefinitionLoadException.java
package io.dws.step.config;

/** Raised when the pod's pinned single-node definition cannot be loaded or is invalid. */
public class DefinitionLoadException extends RuntimeException {

  public DefinitionLoadException(String message) {
    super(message);
  }

  public DefinitionLoadException(String message, Throwable cause) {
    super(message, cause);
  }
}

File: dws-flow/src/DefinitionLoadException.cs
namespace Dws.Flow;

/// <summary>Raised when the pod's pinned single-node definition cannot be loaded or is invalid.</summary>
public sealed class DefinitionLoadException : Exception
{
    public DefinitionLoadException(string message) : base(message)
    {
    }

    public DefinitionLoadException(string message, Exception innerException) : base(message, innerException)
    {
    }
}
